### PR TITLE
Mono fix

### PR DESCRIPTION
**Pull request by danke-sra**

### CODE REVIEW

**Comment by thomasflynn** [comment]
@chenchao1407 does the code look correct to you?  It involves getting the EyeBuffer parameters.  I can cherry-pick and test it, i'd just like your verification that he's getting the correct settings.

**Comment by thomasflynn** [comment]
ok, verified the fix works as expected.

File: GVRf/Framework/src/org/gearvrf/GVRLensInfo.java [hunk 1]
```diff
@@ -80,7 +100,20 @@ public int getFBOWidth() {
      * @return current distortion FBO height value
      */
     public int getFBOHeight() {
-        return mAppSettings.getEyeBufferParms().getResolution();
+        if (!mCustomFBOSize)
+            return mFBOWidth = mAppSettings.getEyeBufferParms().getResolution() *
+                    mAppSettings.getEyeBufferParms().getWidthScale();
```
Comment (chenchao1407) [inline GVRf/Framework/src/org/gearvrf/GVRLensInfo.java]: Here you are returning mFBOWidth for getFBOHeight?

**Comment by chenchao1407** [comment]
There is a typo. And given the fact that this should be only for mono but the fix will also potentially affect VR mode which may cause unpredictable result. My advice is like you can add parameters into VrAppSettings.MonoModeParms to do kinds of restriction.

**Comment by chenchao1407** [comment]
Just double checked the patch. Seems I made a mistake previously. Will verify that tmr morning.

**Comment by chenchao1407** [comment]
@danke-sra Hi Danke, can you help rebase this pull request?

Thanks

**Comment by danke-sra** [comment]
Rebased

**Pull request merged**